lint-python/action.yml

```diff
@@ -1,20 +1,41 @@
 name: Lint Python
 description: Linting for Python projects
+
 inputs:
   exclude_trailing_comma_path:
     required: false
     description: Ignore trailing comma changes in given path
     default: ''
+  src:
```

Comment:
I would call this `working_directory` or something like this.

Reply:
Updated to `working_directory` and the default value to `${{ github.workspace }}`.